srcs/parsing/parsing_utils.c

```diff
@@ -8,4 +8,14 @@ char	*skip_space(char *line)
 	while (line[i] && line[i] == SPACE)
 		i++;; //faire la fonction
 	return (line + i);
-}
\ No newline at end of file
+}
+
+int		get_split_len(char **split)

```

Comment:
this is a very specific name for such a generic function. You're basically getting the length of a null terminated array. I'd call it arrlen_nullterm for example

Reply:
get_array_len ?

Reply:
I'd add something which tells you that the array needs to be null terminated, but I guess it's also kinda obvious from the fact that we're not giving the array size as parameter

Reply:
yeah... I'd prefer not to. Let's just assume all our arrays are null terminated. Else, how do you even calculate how many you got ? haha